xbmc/Application.cpp

```diff
@@ -1873,12 +1873,9 @@ void CApplication::Render()
   if(!g_Windowing.BeginRender())
     return;
 
-  CDirtyRegionList dirtyRegions;
-
   // render gui layer
   if (!m_skipGuiRender)
   {
-    dirtyRegions = g_windowManager.GetDirty();
```

Comment:
Actually I think GetDirty can be removed.
GetDirty returns ``m_tracker.GetDirtyRegions()``
Seems the code that uses GetDirtyRegions is now:
https://github.com/xbmc/xbmc/blob/master/xbmc/guilib/GUIWindowManager.cpp#L1094